docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md

```diff
@@ -4,42 +4,123 @@ displayed_sidebar: "English"
 
 # SHOW ROUTINE LOAD
 
-## Examples
+import RoutineLoadPrivNote from '../assets/commonMarkdown/RoutineLoadPrivNote.md'
+
+## Description
+
+Shows the execution information of Routine Load jobs.
+
+<RoutineLoadPrivNote />
 
-1. Show all routine import jobs (including stopped or cancelled jobs) with the name test1. The result is one or more rows.
+## Syntax
 
-    ```sql
-    SHOW ALL ROUTINE LOAD FOR test1;
-    ```
+```SQL
+SHOW [ALL] ROUTINE LOAD FOR [<db_name>.]<job_name>
+[ WHERE [ STATE = { "NEED_SCHEDULE" | "RUNNING" | "PAUSED" | "UNSTABLE" | "STOPPED" | "CANCELLED"  } ] ]
+[ ORDER BY field_name [ ASC | DESC ] ]
+[ LIMIT { [offset, ] limit | limit OFFSET offset } ]
+```
 
-2. Show the currently running routine import job with the name test1
+:::NOTE
 
-    ```sql
-    SHOW ROUTINE LOAD FOR test1;
-    ```
+You can add the `\G` option to the statement (such as `SHOW ROUTINE LOAD FOR <job_name>\G`) to vertically display the return result rather than in the usual horizontal table format.
 
-3. Show all routine import jobs (including stopped or cancelled jobs) in example_ db. The result is one or more rows.
+:::
 
-    ```sql
-    use example_db;
-    SHOW ALL ROUTINE LOAD;
-    ```
+## Parameters
 
-4. Show all running routine import jobs in example_ db
+| **Parameter**                     | **Required** | **Description**                                              |
+| --------------------------------- | ------------ | ------------------------------------------------------------ |
+| db_name                           | No           | The name of the database to which the load job belongs.      |
+| job_name                          | Yes          | The name of the load job.                                    |
+| ALL                               | No           | Displays all load jobs, including those in the `STOPPED` or  `CANCELLED` states. |
+| STATE                             | No           | State of the load job.                                       |
+| ORDER BY field_name [ASC \| DESC] | No           | Sorts the return result in ascending or descending order based on the specified field. The following fields are supported: `Id`, `Name`, `CreateTime`, `PauseTime`, `EndTime`, `TableName`, `State`, and `CurrentTaskNum`.To sort the return result in ascending order, specify `ORDER BY field_name ASC`.To sort the return result in descending order, specify `ORDER BY field_name DESC`.If you do not specify the field or the sort order, the return result is sorted in ascending order of `Id` by default. |
+| LIMIT limit                       | No           | The number of load jobs that will be returned. If this parameter is not specified, the information of all load jobs that match the filter conditions are displayed. For example, if `LIMIT 10` is specified, only the information of 10 load jobs that match filter conditions will be returned. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/df57c136-23b4-48a2-9c50-9a56c3537005)
This sentence should be moved forward preceding "If this parameter is not specified, the information of all load jobs that match the filter conditions are displayed." to make the flow of the parameter description more logical.